src/SpotifyApi.NetCore/PlaylistsApi.cs

```diff
@@ -194,5 +196,132 @@ public async Task<T> GetPlaylist<T>(string playlistId, string accessToken = null
+        /// <param name="accessToken">The bearer token which is gotten during the authentication/authorization process.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PlaylistSnapshotID"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/playlists/add-tracks-to-playlist/
+        /// </remarks>
+        public async Task<T> AddItemsToPlaylist<T>(
+            string playlistId,
+            string[] uris,
+            int? position = null,
+            string accessToken = null
+            )
+        {
+            if (string.IsNullOrWhiteSpace(playlistId)) throw new
+                    ArgumentException("A valid Spotify playlist id must be specified.");
+
+            if (uris?.Length < 1 || uris?.Length > 100) throw new
+                     ArgumentException("A minimum of 1 and a maximum of 100 Spotify uri must be specified.");
+
+            if (position != null && position < 0) throw new
+                     ArgumentException("The position if supplied has to be 0 or greater than 0.");
+
+            var builder = new UriBuilder($"{BaseUrl}/playlists/{playlistId}/tracks");
+            return (await Post<T>(builder.Uri, new { uris, position }, accessToken)).Data;
+        }
+
+        #endregion
+
+        #region ChangePlaylistDetails
+        class PutDataPayloadPlaylistDetails
+        {
+            [JsonProperty("name")]
+            public string Name { get; set; }
+            [JsonProperty("public")]
+            public bool? Public { get; set; }
+            [JsonProperty("collaborative")]
+            public bool? Collaborative { get; set; }
+            [JsonProperty("description")]
+            public string Description { get; set; }
+
+            public PutDataPayloadPlaylistDetails(string name, bool? makePublic, bool? collaborative, string description)
+            {
+                Name = name;
+                Public = makePublic;
+                Collaborative = collaborative;
+                Description = description;
+            }
+
+            public bool ShouldSerializeName()
+            {
+                return !string.IsNullOrEmpty(Name);
+            }
+
+            public bool ShouldSerializePublic()
+            {
+                return Public != null;
+            }
+
+            public bool ShouldSerializeCollaborative ()
+            {
+                return Collaborative != null;
+            }
+
+            public bool ShouldSerializeDescription()
+            {
+                return !string.IsNullOrEmpty(Description);
+            }
```

Comment:
Don't do this. See https://github.com/Ringobot/SpotifyApi.NetCore/pull/40/files#diff-0980c5ae9847c84bd559259bfae5ab7eR7